openerp/models.py

```diff
@@ -371,21 +371,34 @@ def _field_create(self, cr, context=None):
         """
         if context is None:
             context = {}
+
+        module = context.get('module')
+
+        # NOTE: this variable is also initialised by __init__ of ir.model.data
+        #       but it may not be already initialized. For the same reason, we
+        #       fill the variable directly instead of calling _update_dummy()
+        if getattr(self.pool, 'model_data_reference_ids', None) is None:
+            self.pool.model_data_reference_ids = {}
+
```

Comment:
A reference is made in `ir.model` as `loads`: https://github.com/odoo/odoo/blob/master/openerp/addons/base/ir/ir_model.py#L903

then used in `_process_end` to delete old xmlids: https://github.com/odoo/odoo/blob/master/openerp/addons/base/ir/ir_model.py#L1234